pkg/test/env_test.go

```diff
@@ -138,7 +138,7 @@ func TestUnsetEnvVarAndRestore(t *testing.T) {
 }
 
 func unsetVariable(t *testing.T) string {
-	u, err := uuid.NewV4()
+	u, err := uuid.NewRandom()
```

Comment:
I used `NewRandom()` so as to not disturb the test, as it returns `(uuid,err)`, while `uuid.New()` return `(uuid)`
`uuid.New()` is equivalent to `uuid.Must(uuid.NewRandom()) `
hope it helps?

Reply:
but is it a problem to drop the error and require the generation of the UUID pass? I mean, do you expect that it would fail?
Personally, I would vote for `uuid.New()` to make the test a bit simpler and not deal with the error.

Reply:
No it's not a problem, I can change it now.

Reply:
done in 96f0276